src/test/java/org/folio/oaipmh/processors/OaiPmhJsonParserTest.java

```diff
@@ -30,21 +34,36 @@ void testParserHandleValidData() {
   }
 
   @Test
-  @Disabled("Temporary disable")
+  void testParserHandleInternalError() {
+    var instances = "{  \"instanceId\": \"e6bc03c6-c137-4221-b679-a7c5c31f986c\" , \"source\": \"FOLIO\", " +
+      "\"updatedDate\": \"2020-06-15T11:07:48.563Z\",  \"deleted\": \"false\",  \"suppressFromDiscovery\": \"false\"}" +
+      "\n\r { \"instanceId\" : \"a7bc91c6-c137-4221-b679-a7c5c31f986c\", \"source\": \"FOLIO\"," +
+      " \"updatedDate\": \"2020-06-15T11:07:48.732Z\",  \"deleted\": \"false\",  \"suppressFromDiscovery\": \"false\"}";
+
```

Comment:
Is the value of this variable exactly the same or nearly identical in the next test? If it is, it might be better to extract it into a separate variable or create a private method to build the JSON value